src/components/posts/PostCommentListItemForm.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { useState } from 'react'
+import { Box } from '@mui/material'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
+import client from '../../utils/client'
+
+function CommentForm({ post, onCommentAdded }) {
+  const [comment, setComment] = useState('')
+
+  const createComment = (event, postId) => {
+    event.preventDefault()
+    client
+      .post(`/post/${postId}/comment`, { content: comment })
+      .then((res) => {
+        setComment('')
+        onCommentAdded(post, res.data.data.comment)
+      })
+      .catch((err) => {
+        console.log(err.message)
+      })
+  }
+
+  const handleComment = (event) => {
+    event.preventDefault()
+    const { value } = event.target
+    setComment(value)
+  }
+
+  return (
+    <>
+      {post.id && (
```

Comment:
Same as above comment. It would be easier to read if we could bring this conditional render outside of the main return, 

So if not `post.id` return empty fragment